modules/navigation.js

```diff
@@ -0,0 +1,23 @@
+export default function showSection(sectionName) {
+  const listDiv = document.querySelector('.list');
+  const formDiv = document.getElementById('form');
+  const contactDiv = document.querySelector('.contact');
+
+  listDiv.classList.add('hide-seek');
+  formDiv.classList.add('hide-seek');
+  contactDiv.classList.add('hide-seek');
+
+  switch (sectionName) {
+    case 'list':
+      listDiv.classList.remove('hide-seek');
+      break;
+    case 'form':
+      formDiv.classList.remove('hide-seek');
+      break;
+    case 'contact':
+      contactDiv.classList.remove('hide-seek');
+      break;
+    default:
+      break;
+  }
+}
```

Comment:
- [ ] Kindly change this to an arrow function in keeping with the ES6 syntax requirement.